client/my-sites/plugins/plugin-management-v2/plugin-row-formatter/index.tsx

```diff
@@ -92,8 +91,11 @@ export default function PluginRowFormatter( {
 			selectedSite && isPluginActionInProgress( state, selectedSite.ID, pluginId, INSTALL_PLUGIN )
 	);
 
+	const { activation, autoupdate } = useSelector( ( state ) =>
```

Comment:
These `useSelector` are crucial to avoid re-renders. When we use `const state = useSelector( ( state ) => state );` we're re-rendering that component on any change of the state 🤯

Reply:
🤯  We use selectors like that in a lot of places, the warning is always shown in the browser but ignored.

Reply:
Yeah, this works "okay" for screens with fewer elements, but here we feel how much difference that does!